### DISCUSSION TITLE
Add universal search to Lawnchair

### DISCUSSION THREAD

**Discussion by amin192022** [LawnchairLauncher/lawnchair#3285] [Ideas]
Entering a query there will bring up both web and phone content, including applications, app shortcuts, contacts, Play Store results, and screenshots
![InShot_20230126_153624988](https://user-images.githubusercontent.com/113847974/214865121-a375e47a-6d5e-4598-95bb-78a9b9e03fa1.jpg)

**Comment by whoisabdi** [comment]
this can be closed now (?)

**Comment by SuperDragonXD** [comment]
Hello! I know that this reply is a bit late, but as stated by 0xAbdiiii, we now support global search in Lawnchair 14+.
